packages/core/src/Stores/traders-hub-store.js

```diff
@@ -868,4 +845,19 @@ export default class TradersHubStore extends BaseStore {
     setIsSetupRealAccountOrGoToDemoModalVisible(value) {
         this.is_setup_real_account_or_go_to_demo_modal_visible = value;
     }
+
+    setTradingPlatformAvailability() {
+        this.root_store.client.trading_platform_available_accounts.forEach(account => {
+            if (account.product === 'synthetic') {
+                // `trading_platform_available_accounts` does not have an entry for 'standard' accounts,
+                // so adding 'standard' sub-account type to the available accounts for synthetic products.
+                this.available_accounts[account.sub_account_type] = true;
```

Comment:
We do not need to make this complicated. `sub_account_type` and `sub_account_category` should not be a concern for the FE team. All we need to care about is the `product` and this needs to be fixed from their side.